src/DssSpell.sol

```diff
@@ -69,202 +77,175 @@ contract DssSpellAction is DssAction {
+        // Atlas: https://sky-atlas.powerhouse.io/A.4.1.2.1.4.2.5_Set_Conversion_Fee_In_MKR_To_SKY_Conversion_Contract_To_1%25/1f1f2ff0-8d73-804c-948b-fddc869fcb65%7Cb341f4c0b83472dc1f9e1a3b
 
-        // Execute Spark proxy spell at 0xe7782847eF825FF37662Ef2F426f2D8c5D904121
-        ProxyLike(SPARK_PROXY).exec(SPARK_SPELL, abi.encodeWithSignature("execute()"));
-    }
+        // File 1% fee on MKR_SKY
+        DssExecLib.setValue(MKR_SKY, "fee", 1 * WAD / 100);
 
-    // ---------- Helper Functions ----------
+        // ---------- Replace Nova Operator ----------
+        // Forum: https://forum.sky.money/t/technical-scope-of-the-nova-allocator-adjustment/27175
+        // Forum: https://forum.sky.money/t/technical-scope-of-the-nova-allocator-adjustment/27175/2
+        // Poll: https://vote.sky.money/polling/QmYt7nbx
 
-    /// @notice Returns the global debt ceiling using the governance agreed formula
-    /// @notice sum(max(debt, line)) for all other ilks + max(LSEV2-SKY-A debt, stUSDS BEAM maxLine)
-    function _calculateLine() internal view returns (uint256 _line) {
-        bytes32[] memory ilks = IlkRegistryAbstract(ILK_REGISTRY).list();
+        // AllocatorBufferLike(ALLOCATOR_NOVA_A_BUFFER).approve(USDS, NOVA_OPERATOR, 0);
+        AllocatorBufferLike(ALLOCATOR_NOVA_A_BUFFER).approve(USDS, NOVA_OPERATOR, 0);
 
-        for (uint256 i; i < ilks.length; i++) {
-            (uint256 Art, uint256 rate,, uint256 line,) = VatAbstract(MCD_VAT).ilks(ilks[i]);
+        // AllocatorRolesLike(ALLOCATOR_ROLES).setUserRole("ALLOCATOR-NOVA-A", NOVA_OPERATOR, 0, false);
+        AllocatorRolesLike(ALLOCATOR_ROLES).setUserRole("ALLOCATOR-NOVA-A", NOVA_OPERATOR, 0, false);
 
-            if (ilks[i] == "LSEV2-SKY-A") {
-                _line += _max(Art * rate, RateSetterLike(STUSDS_RATE_SETTER).maxLine());
-            } else {
-                _line += _max(Art * rate, line);
-            }
-        }
-    }
+        // AllocatorRolesLike(ALLOCATOR_ROLES).setRoleAction("ALLOCATOR-NOVA-A", 0, ALLOCATOR_NOVA_A_VAULT, VaultLike.draw.selector, false);
+        AllocatorRolesLike(ALLOCATOR_ROLES).setRoleAction("ALLOCATOR-NOVA-A", 0, ALLOCATOR_NOVA_A_VAULT, VaultLike.draw.selector, false);
+
+        // AllocatorRolesLike(ALLOCATOR_ROLES).setRoleAction("ALLOCATOR-NOVA-A", 0, ALLOCATOR_NOVA_A_VAULT, VaultLike.wipe.selector, false);
+        AllocatorRolesLike(ALLOCATOR_ROLES).setRoleAction("ALLOCATOR-NOVA-A", 0, ALLOCATOR_NOVA_A_VAULT, VaultLike.wipe.selector, false);
+
+        // AllocatorRolesLike(ALLOCATOR_ROLES).setIlkAdmin("ALLOCATOR-NOVA-A", NOVA_PROXY);
+        AllocatorRolesLike(ALLOCATOR_ROLES).setIlkAdmin("ALLOCATOR-NOVA-A", NOVA_PROXY);
+
+        // WardsAbstract(ALLOCATOR_NOVA_A_VAULT).rely(NOVA_PROXY);
+        WardsAbstract(ALLOCATOR_NOVA_A_VAULT).rely(NOVA_PROXY);
+
+        // WardsAbstract(ALLOCATOR_NOVA_A_VAULT).deny(MCD_PAUSE_PROXY);
+        WardsAbstract(ALLOCATOR_NOVA_A_VAULT).deny(MCD_PAUSE_PROXY);
```

Comment:
Objectively, it means the same, but semantically we usually refer to the `PAUSE_PROXY` as `address(this)`:
```suggestion
        WardsAbstract(ALLOCATOR_NOVA_A_VAULT).deny(address(this));
```